test/pub_test.dart

```diff
@@ -149,7 +139,7 @@ dev_dependencies:
 PSDependency findDependency(PSDependencyList deps, {String name}) =>
     deps.firstWhere((dep) => dep.name.text == name, orElse: () => null);
 
-testDepListContains(
+void testDepListContains(
```

Comment:
if it returns nothing, explicit is gold! The tools love it. Why do you hate our tools so? :stuck_out_tongue_winking_eye: